types/howler/index.d.ts

```diff
@@ -248,29 +238,26 @@ export class Howl {
     ): this;
     off(event?: string, callback?: HowlCallback | HowlErrorCallback, id?: number): this;
 
-    stereo(pan: number, id?: number): this | void;
-    pos(x: number, y: number, z: number, id?: number): this | void;
-    orientation(x: number, y: number, z: number, xUp: number, yUp: number, zUp: number): this | void;
-    pannerAttr(
-        o: {
-            coneInnerAngle?: number | undefined;
-            coneOuterAngle?: number | undefined;
-            coneOuterGain?: number | undefined;
-            distanceModel: 'inverse' | 'linear';
-            maxDistance: number;
-            panningModel: 'HRTF' | 'equalpower';
-            refDistance: number;
-            rolloffFactor: number;
-        },
-        id?: number,
-    ): this;
+    stereo(): number;
+    stereo(pan: number, id?: number): number | this;
+
+    pos(): SpatialPosition;
+    pos(x: number, y?: number, z?: number, id?: number): this;
+
+    orientation(): SpatialOrientation;
+    orientation(x: number, y?: number, z?: number, id?: number): this;
+
+    pannerAttr(id?: number): PannerAttributes;
+    pannerAttr(options: PannerAttributes, id?: number): this;
 }
 
-export interface Howler {
+export class HowlerGlobal {
```

Comment:
I reviewed the source and it doesn't look like `HowlerGlobal` is exported (though it is made global): https://github.com/goldfire/howler.js/blob/143ae442386c7b42d91a007d0b1f1695528abe64/dist/howler.js#L2565-L2582

One way to achieve this is to do:

```ts
declare global {
  class HowlerGlobal {
    ...
  }
  const Howler: typeof HowlerGlobal;
}

export const Howler: typeof HowlerGlobal;
```